HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -426,6 +426,101 @@ def self.apply_skylights(hpxml)
+
+      if cooling_system_type == HPXML::HVACTypeCentralAirConditioner
+        if not year_installed.nil?
+          cooling_system.cooling_efficiency_seer = HVAC.lookup_hvac_efficiency(year_installed, cooling_system_type, cooling_system_fuel, 'SEER')
+          cooling_system.cooling_efficiency_seer_isdefaulted = true
+        end
+      elsif cooling_system_type == HPXML::HVACTypeRoomAirConditioner
+        if not year_installed.nil?
+          cooling_system.cooling_efficiency_eer = HVAC.lookup_hvac_efficiency(year_installed, cooling_system_type, cooling_system_fuel, 'EER')
+          cooling_system.cooling_efficiency_eer_isdefaulted = true
+        end
+      end
+    end
+
+    hpxml.heat_pumps.each do |heat_pump|
+      # FIXME: double-check the line below
+      next unless ((heat_pump.cooling_efficiency_seer.nil? && heat_pump.cooling_efficiency_eer.nil?) || (heat_pump.heating_efficiency_hspf.nil? && heat_pump.heating_efficiency_cop.nil?))
+
+      year_installed = heat_pump.year_installed
+      heat_pump_type = heat_pump.heat_pump_type
+      heat_pump_fuel = HPXML::FuelTypeElectricity
+
+      if [HPXML::HVACTypeHeatPumpAirToAir, HPXML::HVACTypeHeatPumpMiniSplit].include? heat_pump_type
+        if (not year_installed.nil?) && (not heat_pump.fraction_cool_load_served == 0) && heat_pump.cooling_efficiency_seer.nil?
+          heat_pump.cooling_efficiency_seer = HVAC.lookup_hvac_efficiency(year_installed, heat_pump_type, heat_pump_fuel, 'SEER')
+          heat_pump.cooling_efficiency_seer_isdefaulted = true
+        end
+        if (not year_installed.nil?) && (not heat_pump.fraction_heat_load_served == 0) && heat_pump.heating_efficiency_hspf.nil?
+          heat_pump.heating_efficiency_hspf = HVAC.lookup_hvac_efficiency(year_installed, heat_pump_type, heat_pump_fuel, 'HSPF')
+          heat_pump.heating_efficiency_hspf_isdefaulted = true
+        end
+      end
+
+      # If heat pump has no cooling/heating load served, assign arbitrary value for cooling/heating efficiency value
+      if (heat_pump.fraction_cool_load_served == 0) && heat_pump.cooling_efficiency_seer.nil? && heat_pump.cooling_efficiency_eer.nil?
+        if heat_pump_type == HPXML::HVACTypeHeatPumpGroundToAir
+          heat_pump.cooling_efficiency_eer = 16.6
+          heat_pump.cooling_efficiency_eer_isdefaulted = true
+        else
+          heat_pump.cooling_efficiency_seer = 13.0
+          heat_pump.cooling_efficiency_seer_isdefaulted = true
+        end
+      end
+      if (heat_pump.fraction_heat_load_served == 0) && heat_pump.heating_efficiency_hspf.nil? && heat_pump.heating_efficiency_cop.nil?
+        if heat_pump_type == HPXML::HVACTypeHeatPumpGroundToAir
+          heat_pump.heating_efficiency_cop = 3.6
+          heat_pump.heating_efficiency_cop_isdefaulted = true
+        else
+          heat_pump.heating_efficiency_hspf = 7.7
+          heat_pump.heating_efficiency_hspf_isdefaulted = true
+        end
+      end
```

Comment:
I wanted to assign an arbitrary value for heating/cooling efficiency if the heat pump has no cooling/heating load served. I guess I can simply assign default heating/cooling efficiency regardless of heating/cooling load served. This will be removed.